BUILD.bazel

```diff
@@ -499,6 +499,7 @@ cc_library(
         ":spirv_tools_link",
         ":test_lib",
         "@effcee//:effcee",
+        "@com_googlesource_code_re2//:re2",
```

Comment:
Can this be `@re2//:re2`? This will get it to use the module instead of the workspace.